src/scss/abstracts/_variables.scss

```diff
@@ -1 +1,2 @@
-$color-background: #202020;
\ No newline at end of file
+$color-background: #202020;
+$color-style: #29abe2;
```

Comment:
Use a meaningful name for this variable

Reply:
this is the major style color used through out the page so this seem more related.